src/ActiveXCore/AXDOM/Element.cpp

```diff
@@ -22,7 +22,7 @@ Element::Element(const FB::JSObjectPtr& element, IWebBrowser *web)
       m_axDisp(std::dynamic_pointer_cast<IDispatchAPI>(element)->getIDispatch()), m_webBrowser(web)
 {
     if (!m_axDisp)
-        throw std::bad_cast("This is not a valid object");
+        throw std::bad_cast();
```

Comment:
I would be a lot happier if instead of removing the messages we could instead subclass bad_cast and provide a version which does allow messages.  This doesn't seem like it would be an onerous task and it would prevent us from losing useful diagnostic information in some cases.